packages/runtime/src/runtime.ts

```diff
@@ -572,6 +573,24 @@ export class Runtime {
     return lsig;
   }
 
+  /**
+   * Transfers `amount` of microAlgos from `from` address to `to` address
+   * @param from From account
+   * @param to to address
+   * @param amount amount of algo in microalgos
+   */
+  fundAccount (from: RuntimeAccount, to: string, amount: number): void {
```

Comment:
```suggestion
  fundAccount (from: RuntimeAccount, to: AccountAddress, amount: number): void {
```
I think we have a type `AccountAddress = string`